OpenSim/Tests/Wrapping/testWrapping.cpp

```diff
@@ -66,6 +67,13 @@ int main()
         std::cout << "Exception: " << e.what() << std::endl;
         failures.push_back("TestShoulderModel (multiple wrap)"); }
 
+    try{
+        testWrapObjectUpdateFromXMLNode30515();
+    } catch (const std::exception& e) {
```

Comment:
I assume you looked at that visually to make sure WrapObjects still show with the right shape/location/scale etc. to make sure the changes to the XML representation didn't have other unintended consequences.